components/Chat/ChatSettings/ChatSettingsPanel/ChatGeneralSettingsPanel.vue

```diff
@@ -222,3 +264,17 @@
     },
   };
 </script>
+
+<style lang="scss">
+  #settings > * {
+    height: 45px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    margin: 0 0 0 0;
+    align-self: center;
+  }
+  #divider {
+    margin: 11.25px 0 11.25px 0;
+    padding: 0 0 0 0;
```

Comment:
`padding: 0 0 0 0;` can be replaced with the class `pa-0`

Reply:
The idea was to have a single rule that does everything, instead of copy-pasting classes. It makes it more obvious what everything does when reading later on.